docs/configuring/enabling-nio.html

```diff
@@ -119,12 +124,21 @@ <h2 class="heading-2">Using Kitura-NIO</h2>
 </pre></code>
         <p>Using the environment variable we make sure that only one out of Kitura-NIO and Kitura-Net is linked into your application's binary.</p>
         <p>Please note that though Kitura-NIO has its own GitHub repository, the package name is KituraNet. This is because Kitura-NIO and Kitura-net are expected to provide identical APIs, therefore it makes sense if they share the package name too.</p>
+        <h2 class="heading-2">Using Kitura-WebSocket-NIO</h2>
+        <p><a href="https://github.com/IBM-Swift/Kitura-WebSocket-NIO">Kitura-WebSocket-NIO</a> and <a href="https://github.com/IBM-Swift/Kitura-WebSocket">Kitura-WebSocket</a> have seperate GitHub repositories, but an identical API.  The underlying code, however is different in how it functions, so it is often helpful to include this snippet of code in your <strong>Package.swift</strong> file so that the correct package is used:</p>
```

Comment:
separate!

Reply:
"The underlying code however functions differently, so it can be helpful to include the following snippet of code in your <strong>Package.swift</strong> file to ensure that the correct package is used:"